toolkit/tools/internal/testutils/testrpms/download-test-utils.sh

```diff
@@ -22,23 +22,39 @@ do
             echo "Usage: download-test-utils.sh [-t IMAGE_VERSION] [-s IMAGE_CREATOR]"
             echo ""
             echo "Args:"
-            echo "  -t IMAGE_VERSION   The Azure Image version to download the RPMs for."
+            echo "  -t IMAGE_VERSION   The image version to download the RPMs for (2.0, 3.0 for Azure Linux or 42 for Fedora)."
             echo "  -s IMAGE_CREATOR   If set to true, the script will create a tar.gz file with the tools and download the rpms needed to test imagecreator."
             echo "  -h Show help"
             exit 1;;
     esac
 done
 
+# Determine the tools file name based on the image version
+BUILD_DIR="$SCRIPT_DIR/build"
+mkdir -p "$BUILD_DIR"
+
+case "${IMAGE_VERSION}" in
```

Comment:
Doing a switch on just the distro version is not ok. It should be either on distro or distro+version.